portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<Comments> comments = new ArrayList<>();
+  private ArrayList<String> commentString = new ArrayList<>();
+
+  /**
+   * Converts an ArrayList instance into a JSON string using the Gson library.
+   */
+  private String convertToJsonUsingGson() {
+    Gson gson = new Gson();
+    String json = gson.toJson(commentString);
+    return json;
```

Comment:
This is just a personal style issue, but I would forgo the "json" variable and just have
   return gson.toJson(commentString);

This isn't anything you need to change.